db/migrate/20201020184453_add_username_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsernameToUser < ActiveRecord::Migration[5.2]
+  def change
+        add_column :users, :username, :string, null: false
+  end
```

Comment:
Looks like line 3 may have been indented twice.  Let's shift that back to the left so its indented only 2 spaces within line 2

Reply:
We made this change.